cli/lib/kontena/cli/services/secrets/link_command.rb

```diff
@@ -12,13 +12,15 @@ class LinkCommand < Clamp::Command
     def execute
       require_api_url
       token = require_token
-      result = client(token).get("services/#{parse_service_id(name)}")
-      secrets = result['secrets']
-      secrets << parse_secrets([secret])[0]
-      data = {
-        secrets: secrets
-      }
-      client(token).put("services/#{parse_service_id(name)}", data)
+      ShellSpinner "linking #{secret.colorize(:cyan)} from Vault to #{name.colorize(:cyan)} " do
+        result = client(token).get("services/#{parse_service_id(name)}")
+        secrets = result['secrets']
+        secrets << parse_secrets([secret])[0]
+        data = {
+          secrets: secrets
+        }
+        client(token).put("services/#{parse_service_id(name)}", data)
```

Comment:
Not maybe part of this PR's scope, but I find it odd that you can link non-existing secrets..?